paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/TopContainerState.kt

```diff
@@ -0,0 +1,13 @@
+package com.stripe.android.paymentsheet.state
+
+import androidx.annotation.StringRes
+import com.stripe.android.paymentsheet.R
+
+internal data class TopContainerState(
```

Comment:
I’m not a big fan of the name. Maybe `WalletsContainer` or something else?

Reply:
I like the sound of `WalletsContainer`